stage/writing_stage.py

```diff
@@ -0,0 +1,34 @@
+"""Writing to the fuzz target and build script."""
+import logging
+
+from result_classes import Result
+from stage.base_stage import BaseStage
+
+logging.basicConfig(level=logging.DEBUG,
+                    format=('%(asctime)s [PID: %(process)d] %(levelname)s '
+                            '[%(module)s.%(funcName)s]: %(message)s'))
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+
+class WritingStage(BaseStage):
+  """Writing to the fuzz target and build script."""
```

Comment:
can you add some more information on this stage in the docstring? i.e. what are the expected inputs and outputs?

Reply:
Yep, added.